### DISCUSSION TITLE
GitHub Copilot error

### DISCUSSION THREAD

**Discussion by salilrsharma** [skills/.github#312] [Code with Copilot]
Hi, 

After creating the codespace, I am getting the following errors: 

1. GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found. You are currently logged in as <username>"
2. No access to GitHub Copilot found. You are currently logged in as <username>.

As a result, I am unable to continue with the auto completion step.  

Any help would be appreciated. 

Thanks,
Salil

**Comment by sinsukehlab** [comment]
Have you enabled GitHub Copilot in the first place?
See: https://docs.github.com/en/copilot
